examples/minimal-features/nuxt.config.js

```diff
@@ -0,0 +1,35 @@
+export default {
+  loading: false,
+  loadingIndicator: false,
+  fetch: {
+    client: false,
+    server: false
+  },
+  features: {
+    store: false,
+    layouts: false,
+    meta: false,
+    middleware: false,
+    transitions: false,
+    deprecations: false,
+    validate: false,
+    asyncData: false,
+    fetch: false,
+    client: {
+      online: false,
+      prefetch: false,
+      useUrl: true
+    },
+    components: {
+      aliases: false,
```

Comment:
what do you think about flattened options like `compoentAliases`? This way we don't need to always worry about falsy value in the path (Like `components: false`)